ui/src/ReusableComponents/Branding.tsx

```diff
@@ -18,25 +18,20 @@
 import React from 'react';
 import './Branding.scss';
 
-interface BrandingProps{
-    message: string;
-    brand: string;
+function Branding(): JSX.Element {
+    return (
+        <div className="branding-container"
+            aria-label="Powered by Ford Labs">
```

Comment:
Do we want this to always be Ford Labs? Has Sara made a decision on this?

Reply:
Vs what?

Reply:
Realizing this might want to be white-labelable if we're being open source, if we're open source. Or maybe we want it to always be Ford Labs, even if somebody's running their own instance? Not something I'd want to address on this story aside from writing another story

Reply:
We already have ford labs in the code base other places relating to the branding, so this is not new. I just gave it an aria-label. matching the text already present.

Reply:
Yeah, I guess this isn't a this-story specific thing